tests/models/test_cli.py

```diff
@@ -13,16 +13,18 @@
     from io import StringIO
 
 import mlflow
-import mlflow.pyfunc as pyfunc
 import mlflow.sklearn
 from mlflow.utils.file_utils import TempDir
-from mlflow.tracking.utils import path_to_local_file_uri
-from tests.models import test_pyfunc
 
 in_travis = 'TRAVIS' in os.environ
 # NB: for now, windows tests on Travis do not have conda available.
 no_conda = ["--no-conda"] if in_travis and sys.platform == "win32" else []
 
+# NB: need to install mlflow since the pip version does not have mlflow models cli.
+install_mlflow = ["--install-mlflow"] if not no_conda else []
```

Comment:
Speaking of, should install_mlflow throw if it's true but no_conda is true?

Reply:
Maybe. It could be a little annoying though - e.g. switching from conda to no conda right now requires changing only one argument on the command line, you might need to change two. 

I can add a test and throw an exception if you think it's safer / better?